bc_obps/bc_obps/settings.py

```diff
@@ -0,0 +1,131 @@
+"""
+Django settings for bc_obps project.
+
+Generated by 'django-admin startproject' using Django 4.2.9.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.environ.get('SECRET_KEY')
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
```

Comment:
we need to allow anything until we figure out a better way for the cluster
```suggestion
ALLOWED_HOSTS = ['*']
```

Reply:
Added with 932aeed.